development/gcp/cloud-functions/serviceaccountcleaner/README.md

```diff
@@ -0,0 +1,35 @@
+# Cleanup service account secrets using Cloud Function
+
+## Overview
+
+Cloud Function creates a new key for a GCP service account and updates the required secret data. The function is triggered by a  Pub/Sub message sent by a secret stored in Secret Manager.
```

Comment:
You wrote "...and updates the required secret data..." Which secret it updates? Where this secret is stored? Is it a k8s secret?
You wrote "...Pub/Sub message sent by a secret stored in Secret Manager." I would rather say a Secret Manager is sending a pubsub message for a secret stored there not the secret itself. This can confuse a reader.

Reply:
Overview section has been corrected, this function deletes old versions of secrets and corresponding keys.